articles/users/guides/email-verified.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Email Verified Usage
```

Comment:
This is not a Swift article. Are you sure this file belongs to this PR? Maybe a leftover from a rebase?

Reply:
You're right, this file does not belong to this PR. Yes, a leftover from the rebase.

Reply:
It's been corrected now.